### DISCUSSION TITLE
zh/docs/frontend/

### DISCUSSION THREAD

**Discussion by giscus** [casbin/casbin#1295] [Docs comments]
# zh/docs/frontend/

Casbin.js is a Casbin addon that facilitates your access-control management in the frontend application

https://casbin.org/zh/docs/frontend/

**Comment by jayddzjh** [comment]
Is Casbin.js not supported for scenarios where a user has multiple roles, meaning it hasn’t been adapted/implemented yet?

**Reply by hsluoyz** [reply]
@jayddzjh issue created at: https://github.com/casbin/casbin.js/issues/301
